Sources/ArcGISToolkit/Utility/CredentialInputView.swift

```diff
@@ -210,65 +210,58 @@ struct CredentialInputSheetView: View {
     }
     
     var body: some View {
-        GeometryReader { proxy in
-            VStack {
-                VStack(alignment: .center) {
-                    VStack(spacing: 8) {
-                        Text(title)
-                            .font(.title)
-                            .multilineTextAlignment(.center)
-                        Text(message)
-                            .font(.subheadline)
-                            .foregroundColor(.secondary)
-                            .multilineTextAlignment(.center)
-                    }
-                    .padding(.vertical)
-                    VStack {
-                        switch fields {
-                        case .password:
-                            passwordTextField
-                        case .usernamePassword:
-                            usernameTextField
-                            Divider()
-                            passwordTextField
-                        }
-                        Divider()
-                    }
-                    .padding([.bottom, .horizontal])
-                    HStack {
-                        Spacer()
-                        Button(role: .cancel) {
-                            cancelAction.handler("", "")
-                        } label: {
-                            Text(cancelAction.title)
-                                .padding(.horizontal)
+        NavigationStack {
+            GeometryReader { proxy in
+                VStack {
+                    VStack(alignment: .center) {
+                        VStack(spacing: 8) {
+                            Text(title)
+                                .font(.title)
+                                .multilineTextAlignment(.center)
+                            Text(message)
+                                .font(.subheadline)
+                                .foregroundColor(.secondary)
+                                .multilineTextAlignment(.center)
                         }
-                        .buttonStyle(.bordered)
-                        Spacer()
-                        Button {
-                            isPresented.wrappedValue = false
-                            continueAction.handler(username, password)
-                        } label: {
-                            Text(continueAction.title)
-                                .padding(.horizontal)
+                        .padding(.vertical)
+                        Form {
```

Comment:
Using a form provides a range of benefits here and it also allows this UI to follow this visionOS design guideline: 
<img width="667" alt="Screenshot 2024-09-18 at 5 16 48 PM" src="https://github.com/user-attachments/assets/89b8e28e-139c-4d6b-9a2b-5466a19b0846">